jobs/ci-run/build/buildjuju.yml

```diff
@@ -12,6 +12,12 @@
       - timestamps
       - build-name:
           name: build-juju-${BUILD_LABEL}
+    properties:
+      - build-discarder:
+          days-to-keep: 30
+          num-to-keep: 50
+          artifact-days-to-keep: 30
```

Comment:
I think we can either omit these, or set it to -1. At least as I read it, the default is to just remove them when removing the other bits. They are only here if you want to remove artifacts before removing logs.

Reply:
`if not empty, artifacts from builds older than this number of days will be deleted, but the logs, history, reports, etc for the build will be kept`

Yeah I think you are right, this might be useful for some jobs, but here we don't need it.